extensions/wikia/CategorySelect/js/CategorySelect.view.js

```diff
@@ -77,7 +77,8 @@
 							controller: 'CategorySelectController',
 							data: {
 								articleId: articleId,
-								categories: $wrapper.data( 'categorySelect' ).getData( '.new' )
+								categories: $wrapper.data( 'categorySelect' ).getData( '.new' ),
+								token: mw.user.tokens.get('editToken')
```

Comment:
Just a sanity check: given this is wrapped in $.nirvana.sendRequest is .done() handling the .fail() case too? If not, probably it's worth adding that explicitly